src/Camt054/Decoder/Message.php

```diff
@@ -19,10 +19,17 @@ public function addRecords(DTO\Message $message, SimpleXMLElement $document): vo
         $notifications = [];
 
         $xmlNotifications = $this->getRootElement($document)->Ntfctn;
+        $fallbackCreationDate = $message->getGroupHeader()->getCreatedOn();
         foreach ($xmlNotifications as $xmlNotification) {
+            $recordDateStr = (string) $xmlNotification->CreDtTm;
+            if ($recordDateStr === '') {
+                $notificationDate = $fallbackCreationDate;
```

Comment:
Can you quote the specification where it says that the mandatory `CreDtTm` on `GrpHdr` _must_ be used as a fallback for the optional `CreDtTm` on `Ntfctn` ?